docs/README.md

```diff
@@ -16,16 +17,18 @@ Groupchat service for PTSS Support, built with Go and FCM
    ```bash
    go mod tidy
    ```
-3. Create a configuration file:
-   In the root directory, create a `config.yaml` file with the following content as an example:
-   ```yaml
-   port: "8080"
-   database_url: "postgres://user:password@localhost:5432/mydb"
-   allowed_origins:
-     - "http://localhost:3000"
-     - "https://myapp.com"
-   jwt_secret: "my-secret-key"
-   fcm_server_key: "your-fcm-server-key"
+3. Configure environment variables:
+   Create a `.env` file in the root directory of the project using the provided `.env.example` file as a reference. Example:
```

Comment:
.env.example is in configs which also implies it should be in root, please put .env.example then in the same folder as you expect .env to be in.